app/views/sign_up/registrations/new.html.erb

```diff
@@ -2,65 +2,63 @@
 
 <%= render 'shared/sp_alert' %>
 
-<%= render AlertComponent.new(
-      type: :error,
-      class: 'email-invalid-alert margin-bottom-4 display-none',
-      message: t('sign_up.email.invalid_email_alert_head'),
-    ) %>
-
 <h1 class='margin-y-0'><%= t('titles.registrations.new') %></h1>
 
-<div class='margin-bottom-6'>
-  <%= validated_form_for(
-        @register_user_email_form,
-        html: { autocomplete: 'off' },
-        url: sign_up_register_path,
-      ) do |f| %>
-    <%= f.input :email,
-                label: t('forms.registration.labels.email'),
-                required: true,
-                input_html: { autocorrect: 'off', aria: { invalid: false } } %>
-
-    <span class='email-invalid-alert-inline usa-error-message margin-top-neg-3 margin-bottom-3 display-none' role='alert'>
-      <%= t('sign_up.email.invalid_email_alert_inline') %>
-    </span>
+<%= simple_form_for(
+      @register_user_email_form,
+      html: { autocomplete: 'off' },
+      url: sign_up_register_path,
+    ) do |f| %>
+  <%= render ValidatedFieldComponent.new(
+        form: f,
+        name: :email,
+        as: :email,
+        label: t('forms.registration.labels.email'),
+        required: true,
+        input_html: { autocorrect: 'off' },
+        error_messages: {
+          typeMismatch: t('sign_up.email.invalid_email'),
```

Comment:
To [an earlier comment](https://github.com/18F/identity-idp/pull/5619#discussion_r756382428), I wonder if these keys should be expected to be given as snake_case, then converted to camelCase internally in the implementation of the component.

On one hand, it's more Ruby-ish, and also matches a similar behavior for passing other HTML attributes like `data: { foo_bar: 1 }` ➡️ `data-foo-bar="1"`.

On the other hand, it may be valuable and easier to understand these as mapping one-to-one with [the specific `ValidityState` properties](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState).